src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
According to the checklist, you should add fallback fonts. Fallback fonts are used in case the main font doesn't load or is not available on the user's system. You can add a generic font family like 'sans-serif' as a fallback. For example: `font-family: Roboto, Arial, sans-serif;`